magento/magento2
# #27175 Allows the user to enter in zero as a custom price

### Description (*)

When adding a product to the basket , it was not possible to set a custom price of zero.
This was because zero is considered empty by PHP and this didn't pass a condition in the code.
 
### Fixed Issues (if relevant)

1. fixes magento/magento2#27175: Unable to set custom price to 0 within a buy request

### Manual testing scenarios (*)
This can not be tested manually. 
Can only be tested programmatically by first allowing products to be added to the basket with a custom price. And then add a product to the basket with a custom price value of 0. 
See the issue for more information https://github.com/magento/magento2/issues/27175

### Contribution checklist (*)
